### DISCUSSION TITLE
About version 0.8.4

### DISCUSSION THREAD

**Discussion by aisiklar** [vue-leaflet/vue-leaflet#282] [General]
Hi,
First thanks for this library. 
Few weeks ago, I used this library (ver 0.8.3) in a side project with no problem. But recently, for another one, I tried to use this but I got errors. I could not mount it. 
When checking the difference between my current project and the last one, I saw that there was a version upgrade... Therefore, in this current one I installed the previous ver (0.8.3), which in conclusion caused no problem..

My point is, I might be missing something but is there a problem with the ver. 0.8.4? Anyone having similar issue with this new ver? 
Thanks,

**Comment by mikeu** [comment]
Hi @aisiklar , is the issue you're experiencing the same one discussed toward the end of [this thread](https://github.com/vue-leaflet/vue-leaflet/issues/278)? If so, then you'll find some temporary solutions in that discussion, and I'll be working on a fix for it next week. If not, could you please provide more details about the problem you're facing, like a console error or stack trace or repo that demonstrates it? Thanks!
